tendrl/gluster_bridge/flows/add_volume_brick.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+import etcd
+from tendrl.gluster_bridge.atoms.volume.add_brick import AddBrick
+
+
+class AddVolumeBrick(object):
+    def __init__(self, api_job):
+        super(AddVolumeBrick, self).__init__()
```

Comment:
oh ok. will try that. just retained as was there in original code of create volume flow.